keyboards/system76/launch_1/keymaps/default/readme.md

```diff
@@ -0,0 +1,5 @@
+# Default keymap for launch_1
+
+<img src="https://i.imgur.com/7RTC4iV.png" alt="launch_1 Base Layout Image" title="launch_1 Base Layout" width="750"/>
```

Comment:
Please replace this with markdown.

Reply:
GitHub Markdown cannot add custom widths, but I don't care to argue. Expressed the HTML indirectly in Markdown sans the width constraint. It'll look disproportionately blown up on wide screens.